packages/docs/page-config/styles/tailwind/code/configuringFull.ts

```diff
@@ -0,0 +1,8 @@
+// main.*
```

Comment:
Hm. We could get rid of these comments and provide fileName options for block.code.